mne/io/base.py

```diff
@@ -1065,6 +1065,8 @@ def resample(self, sfreq, npad=None, window='boxcar', stim_picks=None,
 
         inst._data = np.concatenate(new_data, axis=1)
         inst.info['sfreq'] = sfreq
+        if inst.info.get('lowpass') is not None:
+            inst.info['lowpass'] = min(inst.info['lowpass'], sfreq / 2.)
```

Comment:
Turns out my changes uncovered a bug with `raw.resample()` where we didn't update `raw.info['lowpass']`, and we should have.